assets/data/simple_add.xml

```diff
@@ -0,0 +1,7 @@
+<additionals>
```

Comment:
rename to charging_station_additional.xml, same to all other variable occurrence.